### PR TITLE
[sdk] update namespace across whole repo

### PR DESCRIPTION
**Pull request by tijyojwad**
as per our discussions, added `claraparabricks::genomeworks` namespace to all modules

Part of #424

### CODE REVIEW

**Comment by tijyojwad** [comment]
@vellamike @ahehn-nv @mimaric @r-mafi - whoever gets a chance to review it first, if you approve can you merge as well? Thanks!

File: common/base/include/claragenomics/logging/logging.hpp [hunk 1]
```diff
@@ -93,7 +93,10 @@
...
 #endif
 
-namespace claragenomics
+namespace claraparabricks
+{
+
+namespace genomeworks
 {
 namespace logging
 {
```
Comment (mimaric) [inline common/base/include/claragenomics/logging/logging.hpp:102]: Do we want to have a new line between namespaces? We did not do it previously.
I'm fine with both, we should just pick one and stick to it

Comment (tijyojwad) [reply common/base/include/claragenomics/logging/logging.hpp:102]: It's not consistent right now. Some files have a new line and some don't. I want to make it consistent with a new line. I'll push a change to fix the older ones too

Comment (tijyojwad) [reply common/base/include/claragenomics/logging/logging.hpp]: Updated

File: common/base/include/claragenomics/utils/cudautils.hpp [hunk 2]
```diff
@@ -147,7 +150,7 @@ std::size_t find_largest_contiguous_device_memory_section();
 /// \brief starts an NVTX range for profiling which stops automatically at the end of the scope
 /// \param varname an arbitrary variable name for the nvtx_range object, which doesn't conflict with other variables in the scope
 /// \param label the label/name of the NVTX range
-#define CGA_NVTX_RANGE(varname, label) ::claragenomics::cudautils::nvtx_range varname(label)
+#define CGA_NVTX_RANGE(varname, label) ::genomeworks::cudautils::nvtx_range varname(label)
```
Comment (ahehn-nv) [inline common/base/include/claragenomics/utils/cudautils.hpp:153]: This should be `::claraparabricks::genomeworks::cudautils::nvtx_range`

Comment (tijyojwad) [reply common/base/include/claragenomics/utils/cudautils.hpp]: good catch! fixed, and added profiling flag to release mode testing

**Pull request merged**